.github/workflows/KaMPKit-Android.yml

```diff
@@ -13,13 +19,8 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - name: Set up JDK
-        uses: actions/setup-java@v1
-        with:
-          java-version: 1.8
-
       - name: Build
-        run: ./gradlew build
+        run: ./gradlew :app:build
```

Comment:
It probably doesn't matter hugely for KaMPKit, but this change means if we ever add a gradle module we need to also remember to update the CI scripts

Reply:
I've been trying to break things up a bit, and I didn't want a full build on shared for this workflow, which would include shared's tests. That also gets triggered, but from a different flow.